@xen-orchestra/web-core/lib/components/table/UiTable.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <table :class="{ 'vertical-border': verticalBorder, error: color === 'error' }" class="ui-table typo p2-regular">
+    <slot />
+  </table>
+</template>
+
+<script lang="ts" setup>
+import { provide } from 'vue'
+
+const props = defineProps<{
+  color?: 'error'
```

Comment:
What do you think of using a prop `hasError` instead of a `color`? I think it would be more meaningful.
```suggestion
  hasError?: boolean
```